bokehjs/src/lib/core/properties.ts

```diff
@@ -175,7 +175,7 @@ export class Any extends Property<any> {}
 
 export class Array extends Property<any[]> {
   valid(value: unknown): boolean {
-    return isArray(value) || value instanceof Float64Array
```

Comment:
This seems like an obvious and correct extension. Probably ought also to have a test.

Reply:
The type of the property is `any[]`, so the original code was already wrong and this change makes keeps it that way. You can try changing this type to `ArrayLike<any>` or `Arrayable<any>`, but it will likely have disastrous consequences. Alternatively you could add another property type that covers typed arrays. However, I'm not sure how a typed array would end up in this context. bokehjs' deserializer is supposed to produce nd arrays, and those don't make sense in contexts where simple arrays are required. In fact, in kinds' `Array` type I didn't include typed array support.

Reply:
Also, there's `isTypedArray()` type query function in `core/util/types` that covers all typed arrays.

Reply:
I don't fully understand your suggestion so I will leave this as-is for now, since as you say it maintains the status quo.